src/train/trainer/__init__.py

```diff
@@ -233,7 +236,7 @@ def save_model_and_metadata(self, node_type, X_test, y_test):
         mae_map = dict()
         item = self.get_basic_metadata(node_type)
         for component in self.energy_components:
-            mae = self.get_mae(node_type, component, X_test, y_test)
+            mae = self.get_mae(node_type, component, X_test_map[component], y_test_map[component])
```

Comment:
is this still needed after #121?

Reply:
I will rebase the PR.

Reply:
Done rebasing. Please confirm it. Thanks.